helm/conjur-app-deploy/values.schema.json

```diff
@@ -0,0 +1,78 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema",
+	"properties": {
+		"appServiceType": {
+			"type": "string"
+		},
+		"app": {
+			"useLoadBalancers": {
+				"type": "boolean"
+			},
+			"image": {
+				"properties": {
+					"repository": {
+						"type": "string"
+					},
+					"tag": {
+						"type": "string"
+					},
+					"pullPolicy": {
+						"type": "string"
```

Comment:
For `pullPolicy`, we can add an enumerated check for valid values:
```
					"pullPolicy": {
						"type": "string",
						"pattern": "^(Always|Never|IfNotPresent)$
					}
```
(Then again, maybe Helm already has this validation and this would be redundant? We can play with this when Issue #266 is implemented)